cli/pipeline/run.go

```diff
@@ -218,9 +218,11 @@ func runPipelineStep(pipeline *PipelineDefinition, step *PipelineDefinitionStep,
 func deleteAndWait(c kubernetes.Interface, podDefinition *PodDefinition, flags *runCmdFlagsStruct) error {
 	pods := clientset.CoreV1().Pods(podDefinition.Namespace)
 	deleting := false
+	var gracePeriod int64
```

Comment:
we never set this as far as I can tell - why not just pass in 0 in the line below?

Reply:
@knaveofdiamonds - this gets set to `0` as the default value for the type.

We can't just pass 0 below because we need a reference, and you cannot reference an uninstantiated variable (ie: you cannot do `&0`).